app/views/AppealConfirmationPage.scala.html

```diff
@@ -63,6 +63,12 @@ <h1 class="govuk-panel__title" id="page-heading">
     <p class="govuk-body govuk-!-padding-top-4">@messages("appealConfirmation.p1")</p>
     <p class="govuk-body">@messages("appealConfirmation.p2", vrn)</p>
 
+    <p class ="govuk-body">
+        <a class="govuk-link" href="@{controllers.routes.ViewAppealDetailsController.onPageLoad().url}" rel="noreferrer noopener" target="_blank">
```

Comment:
Can you give this link an ID like the other links do?
```suggestion
        <a class="govuk-link" href="@{controllers.routes.ViewAppealDetailsController.onPageLoad().url}" rel="noreferrer noopener" target="_blank" id="print-link">
```